script/Ciel/Day 15/15_01A_CIEL15A_1.txt

```diff
@@ -2684,7 +2684,7 @@ With that in mind, do you still plan on stepping into our world?
 [sha:64c755be6d60f606b040164582b1c1286d7c1312]{
 -- Page 156, Offset 41873.
 -- ―――捨てる必要のない、追われる必要もない、豊かな故郷がありながら」
-Despite having a home that you needn't either abandon or oust yourself from?"
+Despite having a home that you needn't abandon nor oust yourself from?"
```

Comment:
Original sounds a slightly more normal to me

Reply:
Hmmm, I could go with either, but the latter is shorter? idk